docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -434,6 +468,21 @@ Creating a dataset from a generator function is also possible.
                                                     NULL, NULL);
     ```
 
+3. Setting `user_data` for the dataset can be done with `ml_train_dataset_set_property()`
+`dataset` does not take the ownership of `user_data`, so be sure to free after use.
```

Comment:
Is D of Dataset capital or small, please use it consistently.
Remove extra spacing
```suggestion
3. Setting `user_data` for the dataset can be done with `ml_train_dataset_set_property()`, the `Dataset` does not take the ownership of `user_data`, so ensure to free it after use:

Reply:
unified to `dataset`.

Reply:
Removed the spacing